lib/rack/param_to_cookie/version.rb

```diff
@@ -1,7 +1,7 @@
 module Rack
   class ParamToCookie
     VERSION_MAJOR = 2
-    VERSION_MINOR = 0
+    VERSION_MINOR = 1
```

Comment:
Yes, I think this would be true no matter what default value we set for `max_value`. I think it's better to have a sensible default so yes this will be a breaking change.